documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -258,23 +258,26 @@ spec:
   vms: <6>
     - id: <source_vm> <7>
     - name: <source_vm>
-      hooks: <8>
+      namespace: {namespace} <8>
+      hooks: <9>
         - hook:
             namespace: {namespace}
-            name: <hook> <9>
-          step: <step> <10>
+            name: <hook> <10>
+          step: <step> <11>
 EOF
 ----
 <1> Specify the name of the `Plan` CR.
 <2> Specify whether the migration is warm or cold. If you specify a warm migration without specifying a value for the `cutover` parameter in the `Migration` manifest, only the precopy stage will run.
 <3> You can add multiple network mappings.
 <4> Specify the name of the `NetworkMap` CR.
 <5> Specify the name of the `StorageMap` CR.
-<6> You can use either the `id` _or_ the `name` parameter to specify the source VMs.
+<6> For all source providers except for {virt}, you can use either the `id` _or_ the `name` parameter to specify the source VMs. +
```

Comment:
`namespace` needs to be specified as well

Reply:
yeah, that's (8)
@RichardHoch I think we need to consider replacing this part with an example per-provider as it starts to become complicated to have many exceptions per-item

Reply:
@ahadas @anarnold97 I think having an example per provider is a good idea. Can it wait until after GA?

Reply:
@RichardHoch - I will have to defer to Arik, but I think the example is a good idea.
Do you think you could get it done in your post-release clean-up to minimize the delay?
Thanks

Reply:
@ahadas @anarnold97 We are talking about this section https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.4/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-from-cli

@ahadas  Did you mean we should have examples per step (that is, all the examples of Secret manifest in step 1, all the the examples of Provider manifests in step 2, etc.) or instructions per provider type (that is RHV manifests from start to finish in one section, vSphere from start to finish in another)? 

In theory, i should be able to create all the manifests from what's written here and have QE check them. 

In reply to another PR, Arik suggested that we reorganize the document by provider, as in...

1. vSphere migrations
  a. UI
  b. CLI
  
2. RHV migrations
 a. UI
 b. CLI 
 
That might be a good long-term idea.

Reply:
> @ahadas Did you mean we should have examples per step (that is, all the examples of Secret manifest in step 1, all the the examples of Provider manifests in step 2, etc.) or instructions per provider type (that is RHV manifests from start to finish in one section, vSphere from start to finish in another)?

instructions per provider type

> 
> In theory, i should be able to create all the manifests from what's written here and have QE check them.
> 
> In reply to another PR, Arik suggested that we reorganize the document by provider, as in...
> 
> 1. vSphere migrations
>    a. UI
>    b. CLI
> 2. RHV migrations
>    a. UI
>    b. CLI

yeah, in those (b) sections